frontend/lib/search/filter_bar_button.dart

```diff
@@ -59,10 +60,24 @@ class _FilterBarButtonState extends State<FilterBarButton>
     const paddingPerElement = 8;
     const minNumberElements = 5;
     var totalWidth = MediaQuery.of(context).size.width;
-    var width = min(
-        80.0,
-        (totalWidth - minNumberElements * paddingPerElement) /
-            minNumberElements);
+
+    // Width allowing at least minNumberElements in one row
+    var smallWidth = min(
+      80.0,
+      (totalWidth - minNumberElements * paddingPerElement) / minNumberElements);
+
+    // At least 65 px are needed to fit all category names
```

Comment:
I'm not sure whether this is specified in px or dp. One at least depends on the DPI. So I'm not sure whether this is a constant.

Reply:
So is it a constant?

Reply:
Should be dp/dip. Not sure if that is a constant.

Reply:
But I tested it on multiple devices now and seems to work fine.